src/module/api/core/Controller.cpp

```diff
@@ -70,17 +70,37 @@ float Controller::getBatteryVoltage()
 
 int Controller::getBrightness()
 {
-  colorSensor.getRawColor(rgb);
-  int luminance = 0.298912 * rgb.r + 0.586611 * rgb.g + 0.114478 * rgb.b;
+  int r, g, b;
+  getRawColor(r, g, b);
+  int luminance = 0.298912 * r + 0.586611 * g + 0.114478 * b;
   return luminance;
 }
 
+int limit_max_min(const int value)
```

Comment:
正直、`limit_max_min`だとわかりずらいと思ったので
```suggestion
int limitRgbValue(const int value)
```
とかどうですか？

Reply:
THX MR. OOIWANE !!